docs/rules-engine/README.md

```diff
@@ -0,0 +1,147 @@
+# Otter Rules Engine
+
+## Introduction
+
+This document describes the various concepts and models that are part of the [Otter rules engine module](https://www.npmjs.com/package/@o3r/rules-engine).
+It serves as an introduction course before you can integrate it within your application following the
+[rules engine implementation guide](./how-to-use/README.md).
+
+As a reminder, the rules engine is a client-based mechanism that evaluates a list of [Rulesets](#ruleset) to define [actions](#action) to
```

Comment:
The `As a reminder` looks weird to me in the document that is supposed to introduce the concept
```suggestion
In a nutshell, the rules engine is a client-based mechanism that evaluates a list of [Rulesets](#ruleset) to define [actions](#action) to
```

Reply:
It is a reminder on concepts covered before in other documentation (the npmjs), so in a nutshell would not fit.
Maybe, 
```suggestion
As covered in the [package documentation](https://www.npmjs.com/package/@o3r/rules-engine), the rules engine is a client-based mechanism that evaluates a list of [Rulesets](#ruleset) to define [actions](#action) to
```
What do you think?